.github/workflows/ci.yml

```diff
@@ -1,7 +1,18 @@
 name: CI
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+  push:
+    branches:
+      - main
```

Comment:
semantic-release supports multiple release workflow and it requires enable ci to run on the target branches, see https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#branches.

Reply:
Good catch, only the default distribution channel is enabled here because supporting other chaneels will lead to much more changes due to some Python ecosystem constraints. For example, `pre-release` need to be compatible with pep440, but I did not find any mature solution for this.

Reply:
Got it